app/javascript/components/patient/Patient.js

```diff
@@ -191,7 +190,9 @@ class Patient extends React.Component {
               <Col sm={10} className="item-group">
                 <div>
                   <b>DOB:</b> <span>{this.props.details.date_of_birth && moment(this.props.details.date_of_birth, 'YYYY-MM-DD').format('MM/DD/YYYY')}</span>
-                  {this.state.isMinor && <span className="text-danger"> (Minor)</span>}
+                  {this.props.details && this.props.details.date_of_birth && isMinor(this.props.details.date_of_birth) && (
```

Comment:
you don't need the `this.props.details` anymore, same with the other instance below